tests/Unit/ProjectCodeTest.php

```diff
@@ -47,6 +47,7 @@ public function testProductionClassesHaveUnitTests()
                 Http\Action\Admin\Talk\IndexAction::class,
                 Http\Action\Reviewer\DashboardAction::class,
                 Http\Action\Reviewer\Speaker\ViewAction::class,
+                Http\Action\Admin\Talk\RateAction::class,
```

Comment:
It looks like this action won't be too hard to unit test, maybe we can add some

Reply:
Ha, I'd prefer to test this when we have extracted the validator into a service, otherwise I'd just be testing the validator mostly.